chapter2/scenarios/03_More_Orcs.cfg

```diff
@@ -381,16 +381,16 @@
     {GENERIC_UNIT 2 (Dwarvish Fighter) 2 20}
     [message]
 	    speaker=Augatol
-	    message=_""Our reinforcements have arrived."
```

Comment:
Note this duplicate double quote causes chapter 2 to not even load, so isn't just a language issue.